### PR TITLE
Migration/ Roles console errors fix

### PR DESCRIPTION
**Pull request by ronronscelestes**
## What/Why

Fixed TabsGroup, Stack and Checkbox causing console errors

### CODE REVIEW

File: packages/core/admin/admin/src/pages/SettingsPage/pages/Roles/EditPage/components/RowLabelWithCheckbox/index.js [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
+import { Row, BaseCheckbox, Text } from '@strapi/parts';
 import upperFirst from 'lodash/upperFirst';
 import PropTypes from 'prop-types';
 import React, { memo } from 'react';
 import { useIntl } from 'react-intl';
 import CollapseLabel from '../CollapseLabel';
 import { firstRowWidth } from '../Permissions/utils/constants';
 
+const CustomCheckbox = styled(BaseCheckbox)`
+  margin-right: ${({ theme }) => theme.spaces[2]};
```
Comment (mfrachet) [inline packages/core/admin/admin/src/pages/SettingsPage/pages/Roles/EditPage/components/RowLabelWithCheckbox/index.js:11]: Could you please wrap the BaseCheckbox with a Box and use padding instead? 🙏🏻

**Review by mfrachet — CHANGES_REQUESTED** [review]
One comment and then you can merge 😊

File: packages/core/admin/admin/src/pages/SettingsPage/pages/Roles/EditPage/components/RowLabelWithCheckbox/index.js (hunk 1 above)
Comment (ronronscelestes) [reply packages/core/admin/admin/src/pages/SettingsPage/pages/Roles/EditPage/components/RowLabelWithCheckbox/index.js:11]: makes sense 👌

File: packages/core/admin/admin/src/pages/SettingsPage/pages/Roles/EditPage/components/Permissions/index.js [hunk 2]
```diff
@@ -108,7 +108,13 @@ const Permissions = forwardRef(({ layout, isFormDisabled, permissions }, ref) =>
         onChangeCollectionTypeGlobalActionCheckbox: handleChangeCollectionTypeGlobalActionCheckbox,
       }}
     >
-      <TabGroup id="tabs">
+      <TabGroup
+        id="tabs"
+        label={formatMessage({
+          id: 'Settings.permissions.users.tabs.label',
+          defaultMessage: 'Tabs Permissions',
```
Comment (soupette) [inline packages/core/admin/admin/src/pages/SettingsPage/pages/Roles/EditPage/components/Permissions/index.js:115]: Are you sure this translation exists?

**Review by soupette — CHANGES_REQUESTED** [review]

**Review by soupette — CHANGES_REQUESTED** [review]
This PR LGTM however can you also use the BaseCheckbox in the ActionRow and SubActionRow components?

**Review by soupette — APPROVED** [review]
LGTM

**Pull request merged**